frappe/email/smtp.py

```diff
@@ -210,7 +210,7 @@ def sess(self):
 				self._sess.ehlo()
 
 			if self.login and self.password:
-				ret = self._sess.login((self.login or ""), (self.password or ""))
+				ret = self._sess.login(str(self.login or ""), str(self.password or ""))
```

Comment:
use `frappe.utils.cstr`

Reply:
That way I'll get `unicode` no matter what. I don't want that. I want bytes in Py 2 i.e. Py 2 str. And I want unicode in Py 3 i.e. Py 3 str.

In both cases I want str